lms/templates/learner_dashboard/_dashboard_navigation_courses.html

```diff
@@ -0,0 +1,8 @@
+<%!
```

Comment:
Need to add `<%page expression_filter="h"/>`. This should fix the quality errors.